apps/api/src/app/import/import.service.ts

```diff
@@ -580,12 +585,13 @@ export class ImportService {
           )
         ) {
           throw new Error(
-            `activities.${index}.currency ("${currency}") does not match with "${assetProfile.currency}" and no exchange rate is available from "${currency}" to "${assetProfile.currency}"`
+            `activities.${index}.symbol("${symbol}").currency ("${currency}") does not match with "${assetProfile.currency}" and no exchange rate is available from "${currency}" to "${assetProfile.currency}."`
```

Comment:
This looks strange 😮

Reply:
ooops, this one was an initial change I had made to identify which item was not being found at Yahoo, but I saw you already had fixed it a few lines before, we can drop this one.